charts/presto/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: presto
 description: The official Helm chart for Presto
 type: application
-version: 0.3.0
+version: 0.3.1
```

```suggestion
version: 0.3.0
```
Please, do not bump the chart version, because it triggers the release workflow. Usually we collect few changes for the next release.

---

@dnskr Thanks for suggestion the changing the path. it is working.

Changes of values.yaml need to commit.

I chose different names for podSecurityContext and securityContext because some properties, such as privileged, only apply at the container level and not at the pod level, ensuring that settings are accurately and appropriately applied.